src/main/java/com/vadmack/mongodbtest/controller/ProjectController.java

```diff
@@ -19,8 +20,11 @@
     private final ProjectService service;
 
     @GetMapping
-    public ResponseEntity<List<ProjectDto>> findAll() {
-        return ResponseEntity.ok(service.findAll());
+    public ResponseEntity<List<ProjectDto>> findList(
+            @RequestParam(value = "filterProjectName", defaultValue = "") String name,
+            @RequestParam Optional<Integer> pageNumber,
```

Comment:
It is better to avoid `Optional` parameter. You can use `@RequestParam(required = false)`